### PR TITLE
VIH-8055 Automate Last Minute Boooking

### PR DESCRIPTION
**Pull request by PrasannaDommalapati**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/VIH-8055

### Change description ###

- Added a scenario for Last minute booking to add a participant and the participants in the waiting room will see a toast alert

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

**Review by FarronF — DISMISSED** [review]
This is great. Would it be worth checking the contents of the toastr? Or checking that the participant list updates?

**Review by alberto-hortiguela — DISMISSED** [review]
LGTM

File: VideoWeb/VideoWeb.AcceptanceTests/Features/LastMinuteBooking.feature [hunk 1]
```diff
@@ -0,0 +1,11 @@
+Feature: LastMinuteBooking
+	As a Participant in the hearing 
+  I would like to see a a toast alert when a new participant is added to the hearing
+  So that all the participants are aware that a new participant has been added
+
+@VIH-8055 @HearingTest @Smoketest-Extended
```
Comment (MichaelKeaneathmcts) [inline VideoWeb/VideoWeb.AcceptanceTests/Features/LastMinuteBooking.feature:6]: Can you change the jira tag to a comment, there is little expectation that we will ever run tests based on a jira tag.

Comment (PrasannaDommalapati) [reply VideoWeb/VideoWeb.AcceptanceTests/Features/LastMinuteBooking.feature]: Done

**Review by MichaelKeaneathmcts — APPROVED** [review]

**Pull request merged**